docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_java_dependencies.adoc

```diff
@@ -1,30 +1,106 @@
 
-* The system builds your application by using community dependencies.
+JBS automates this process as much as possible, making it much less time-consuming to create builds that are build from source in a controlled environment.
 
-* The JVM Build Service analyzes your application and determines the dependencies that the system needs to be rebuilt.
+=== Workflow Description
 
-* The JVM Build Service generates `ArtifactBuild` objects to represent each artifact in your application before determining to which repository the artifacts belong. The JVM Build Service tags from which systems the artifacts derive.
+When using the JVM Build Service the end user workflow is as follows:
 
-* The system analyzes the repository and uses the results to find a build strategy. For example, if the system cannot find the JDK version to use, the system uses all versions to determine the most effective version to use.  
+. The system builds your application by using community dependencies.
 
-* The system attempts to build all the dependencies and stores them in container images in an image registry, and `quay.io` is the default.
+. The JVM Build Service analyzes your application and determines the dependencies that the system needs to rebuild.
 
-* When the system has completed all its builds, you can rebuild your application.
+. The JVM Build Service generates `ArtifactBuild` objects to represent each artifact in your application before determining to which repository the artifacts belong. The JVM Build Service tags from which systems the artifacts derive.
 
-* If all the builds are successful, your application's SBOM displays that none of the dependencies came from third-party repositories. However, you may need to troubleshoot when some dependencies fail to build.
+. The system analyzes the repository and uses the results to find a build strategy. For example, if the system cannot find the JDK version to use, the system uses all versions to determine the most effective version to use.
 
-.*Procedure*
+. The system attempts to build all the dependencies and stores them in container images in an image registry, and `quay.io` is the default.
+
+. When the system has completed all its builds, you can rebuild your application.
+
+. If all the builds are successful, your application's SBOM displays that none of the dependencies came from third-party repositories. However, you may need to troubleshoot when some dependencies fail to build.
+
+== Setup
+
+=== Logging into the Server
+
+All interactions with the JVM Build Service require you to be logged into the OpenShift namespace used for your builds. This can be done by copying the login command from https://registration-service-toolchain-host-operator.apps.stone-prd-host1.wdlc.p1.openshiftapps.com/ .
```

Comment:
With the Red Hat modularization standards, we give even one action step a bullet point. So this section would be:

=== Logging into the Server 

All interactions with the JVM Build Service require you to be logged into the OpenShift namespace used for your builds.

.Procedure
- Copy the login command from: < link >.